test/organization.e2e-spec.ts

```diff
@@ -209,58 +210,155 @@ describe('Organization e2e', () => {
     expect(actual.items[0].name.value).toBe(name);
   });
 
-  it('list view of organizations ASC', async () => {
-    // create a bunch of orgs
-
-    const numOrgs = 2;
-    await Promise.all(
-      times(numOrgs).map(() => createOrganization(app, { name: generate() }))
-    );
-
-    const { organizations } = await app.graphql.query(gql`
-      query {
-        organizations(input: { sort: "name", order: ASC }) {
-          items {
-            ...org
+  it('List of organizations sorted by name to be alphabetical, ignoring case sensitivity. Order: ASCENDING', async () => {
+    await createUser(app, { displayFirstName: 'Tammy' });
+    //Create three projects, each beginning with lower or upper-cases
+    await createOrganization(app, {
+      name: 'an Organization ' + faker.random.uuid(),
+    });
+    await createOrganization(app, {
+      name: 'Another Organization' + faker.random.uuid(),
+    });
+    await createOrganization(app, {
+      name: 'Big Organization' + faker.random.uuid(),
+    });
+    await createOrganization(app, {
+      name: 'big Organization also' + faker.random.uuid(),
+    });
+    const sortBy = 'name';
+    const ascOrder = 'ASC';
+    const { organizations } = await app.graphql.query(
+      gql`
+        query organizations($input: OrganizationListInput!) {
+          organizations(input: $input) {
+            hasMore
+            total
+            items {
+              id
+              name {
+                value
+              }
+            }
           }
         }
+      `,
+      {
+        input: {
+          sort: sortBy,
+          order: ascOrder,
+        },
       }
-      ${fragments.org}
-    `);
-
-    const names = organizations.items.map(
-      (o: { name: { value: string; canEdit: boolean; canRead: boolean } }) =>
-        o.name.value
     );
+    // Set a flag that's going to indicate if the projects are in order
+    let isAscending = true;
+    const items = organizations.items;
+    let index = 0;
+    for (const i of items) {
+      if (index < organizations.items.length - 1) {
+        const currName = i.name.value;
+        const nextName = items[index + 1].name.value;
+        const areEqual = currName.toUpperCase() < nextName.toUpperCase();
+        if (!areEqual) isAscending = false;
+        index++;
+      }
+    }
+    expect(isAscending).toBe(true);
```

Comment:
```suggestion
    const items = organizations.items;
    const sorted = orderBy(items, (org) => org.name.value.toLowerCase(), ['asc']);
    expect(sorted).toEqual(items);
```
https://lodash.com/docs/4.17.15#orderBy

Reply:
awesome! I can put this in for the project tests, too